lib/tasks/unpublish.rake

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+namespace :unpublish do
+  desc "Retire a document on GOV.UK e.g. unpublish:retire_document CONTENT_ID='a-content-id' NOTE='A note'"
+  task retire_document: :environment do
+    raise "Missing CONTENT_ID value" if ENV["CONTENT_ID"].nil?
+    raise "Missing NOTE value" if ENV["NOTE"].nil?
+
+    content_id = ENV["CONTENT_ID"]
+    explanatory_note = ENV["NOTE"]
+    locale = ENV["LOCALE"] || "en"
+
+    document = Document.find_by(content_id: content_id, locale: locale)
+    raise "Document must have a published version before it can be retired" unless document.has_live_version_on_govuk
+
+    DocumentUnpublishingService.new.retire(document, explanatory_note, locale: locale) if document.present?
```

Comment:
I guess you don't need this conditional here as it would exception on `document.has_live_version_on_govuk`. You could kill it off by using `find_by!` in the initial call